src/controllers/project.controller.ts

```diff
@@ -142,26 +138,20 @@ export class ProjectController {
 		this.fetchSessions();
 	};
 
-	public reconnect = () => {
-		this.deployments = undefined;
-		this.loadAndDisplayDeployments(false);
-		this.fetchSessions();
-	};
-
```

Comment:
We need this - it runs when we click re-connect in the project UI after the server disconnects.

Reply:
added reconnect